components/grid/selection/overview.md

```diff
@@ -86,17 +94,21 @@ See how the row selection modes work
 
 ## Notes
 
+## Selection in Template
+
+When using the Grid [Template](https://docs.telerik.com/blazor-ui/components/grid/templates) and you want to stop the Selection from being triggered you should add stopPropagation directive to the element
```

Comment:
"stopPropagation" should be in a code block. Add an example of how to actually do that, because you must show which event you need to stop from propagating.

Reply:
Done